src/Database/Log/QueryLogger.php

```diff
@@ -25,6 +26,34 @@
 class QueryLogger
 {
 
+    use InstanceConfigTrait;
+
+    /**
+     * Default Config
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'threshold' => 0,
+        'filter' => null
+    ];
+
+    /**
+     * Constructor
+     *
+     * The $config array takes the following keys:
+     *
+     * - threshold: Threshold in milliseconds to log only query running slower than the given threshold. Default is 0.
+     * - filter: A callable to filter based on the LoggedQuery object.
+     *
+     * @param array $config Config options
+     * @return void
+     */
+    public function __construct(array $config = [])
```

Comment:
No, it has a method to set it, took me some time to find it when I needed it.

```php
// bootstrap.php
ConnectionManager::get('default')->logger(new SlowQueryLogger());
ConnectionManager::get('test')->logger(new SlowQueryLogger());
```

Is what we do in our app. I enable the threshold as needed.